src/app/app.service.ts

```diff
@@ -0,0 +1,46 @@
+import { Injectable } from '@angular/core';
+import * as AWS from 'aws-sdk';
+
+@Injectable()
+export default class AppService {
+  constructor() {
+    this.anonLog();
+  }
+
+  anonLog() {
+    // Configure the credentials provider to use your identity pool
+    AWS.config.region = 'us-east-2'; // Region
+    AWS.config.credentials = new AWS.CognitoIdentityCredentials({
+      IdentityPoolId: 'us-east-2:71bd3c00-e780-43cc-ac49-e1ae9020dd18',
+    });
+    // Make the call to obtain credentials
+    (<AWS.CognitoIdentityCredentials>AWS.config.credentials).get(function () {
+      // Credentials will be available when this function is called.
+      const accessKeyId = AWS.config.credentials.accessKeyId;
+      const secretAccessKey = AWS.config.credentials.secretAccessKey;
+      const sessionToken = AWS.config.credentials.sessionToken;
+    });
+  }
+
+  detectLabels(imageData) {
+    // AWS.region = "us-east-2";
```

Comment:
this is for detect labels. same we we can add detect face in this service. 
Service is returning ES2015 Promise